.dockerignore

```diff
@@ -1,4 +1,3 @@
 vendor
 Dockerfile
-Makefile
```

Comment:
Did you make any changes to the Makefile, that led you to see that the gitignore was ignoring its changes? 
If you made any changes to the Makefile, they have not been commited. But maybe you didn't change it at all. Just wanted to confirm it.